docs/rules/no-deprecated-slot-attribute.md

```diff
@@ -43,16 +43,18 @@ This rule reports deprecated `slot` attribute in Vue.js v2.6.0+.
 ```json
 {
   "vue/no-deprecated-slot-attribute": ["error", {
-    "ignore": ["my-component"]
+    "ignore": ["my-component"],
+    "ignoreParents": ["my-web-component"],
   }]
 }
 ```
 
 - `"ignore"` (`string[]`) An array of tags or regular expression patterns (e.g. `/^custom-/`) that ignore these rules. This option will check both kebab-case and PascalCase versions of the given tag names. Default is empty.
+- `"ignoreParents"` (`string[]`) An array of tags or regular expression patterns (e.g. `/^custom-/`) for parents that ignore these rules. This option is especially useful for Web-Components. This option will check both kebab-case and PascalCase versions of the given tag names. Default is empty.
 
 ### `"ignore": ["my-component"]`
 
-<eslint-code-block fix :rules="{'vue/no-dupe-keys': ['error', {ignore: ['my-component']}]}">
+<eslint-code-block fix :rules="{'vue/no-deprecated-slot-attribute': ['error', {ignore: ['my-component']}]}">
```

Comment:
I also noticed that the rule config is wrong here, if I should move it to a separate PR please let me know.